website/versioned_docs/version-v3.10.x/externaldata.md

```diff
@@ -449,3 +449,71 @@ server := &http.Server{
 ```
 
 2. If `cert-controller` is disabled via the `--disable-cert-rotation` flag, you can use a cluster-wide, well-known CA certificate for Gatekeeper so that your external data provider can trust it without being deployed to the `gatekeeper-system` namespace.
+
+### Authenticate API server against Webhook (Self managed K8s cluster only)
+
+**Note:** To enable authenticate api server requires modifying cluster resources that may not be possible in managed K8s cluster
```

Comment:
```suggestion
**Note:** To enable authenticating the api server you have to be able to modify cluster resources. This may not be possible for managed K8s clusters.
```